controller/src/main/java/io/pravega/controller/server/eventProcessor/SerializedRequestHandler.java

```diff
@@ -0,0 +1,115 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ * <p>
```

Comment:
Please do not change the license header formatting.

Reply:
i think that was because of Ctrl Alt L formatting in idea..  i will be careful not to have them changed in future.